### PR TITLE
BZ2021841: added URLs to tables in point 1 and 4

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.6+ versions.
- [BZ2021841](https://bugzilla.redhat.com/show_bug.cgi?id=2021841)
- [Preview](https://deploy-preview-38689--osdocs.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall)
- @gpei Please verify.

### CODE REVIEW

File: modules/configuring-firewall.adoc [hunk 1]
```diff
@@ -92,11 +96,23 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
...
 
-|Azure
+.4+|Azure
 |`management.azure.com`
 |443, 80
 |Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API Reference] in the Azure documentation to determine the endpoints to allow for your APIs.
 
+|`*.blob.core.windows.net`
+|443, 80
+|Required to download ignition files.
```
Comment (bobfuru) [inline modules/configuring-firewall.adoc:106]: Ignition is a productized utility so we should capitalize it ("Ignition files").

File: modules/configuring-firewall.adoc [hunk 2]
```diff
@@ -92,11 +96,23 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
...
+
+|`login.microsoftonline.com`
+|443, 80
+|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API Reference] in the Azure documentation to determine the endpoints to allow for your APIs.
+
+|`www.office.com`
+|443, 80
+|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API Reference] in the Azure documentation to determine the endpoints to allow for your APIs.
```
Comment (bobfuru) [inline modules/configuring-firewall.adoc:114]:
```suggestion
|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API reference] in the Azure documentation to determine the endpoints to allow for your APIs.
```
^^to match their doc title

**Review by bobfuru — COMMENTED** [review]
Two small nits and otherwise LGTM.

File: modules/configuring-firewall.adoc (hunk 2 above)
Comment (Amrita42) [reply modules/configuring-firewall.adoc:114]: Replaced three instances of `Reference`

File: modules/configuring-firewall.adoc [hunk 3]
```diff
@@ -92,10 +96,22 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
...
+
+|`*.blob.core.windows.net`
+|443, 80
+|Required to download Ignition files.
+
+|`login.microsoftonline.com`
+|443, 80
+|Required to access Azure services and resources. Review the link:https://docs.microsoft.com/en-us/rest/api/azure/[Azure REST API reference] in the Azure documentation to determine the endpoints to allow for your APIs.
+
+|`www.office.com`
```
Comment (gpei) [inline modules/configuring-firewall.adoc:112]: This is a full URL list I found on a proxy server which is used for a fresh Azure cluster installation:

```
api.openshift.com:443
canary-openshift-ingress-canary.apps.gpei-13.qe.azure.devcluster.openshift.com:443
cdn02.quay.io:443
ci-dv2np-image-registry-us-east-1-aunteqmixxpqypvdqwbmjbiloeix.s3.dualstack.us-east-1.amazonaws.com:443
cloud.redhat.com:443
console-openshift-console.apps.gpei-13.qe.azure.devcluster.openshift.com:443
http://168.63.129.16/?
imageregistrygpei1358nqx.blob.core.windows.net:443
infogw.api.openshift.com:443
login.microsoftonline.com:443
management.azure.com:443
oauth-openshift.apps.gpei-13.qe.azure.devcluster.openshift.com:443
quay.io:443
registry.access.redhat.com:443
registry.ci.openshift.org:443
registry.redhat.io:443
```

The one which is related to Azure maybe:
```
login.microsoftonline.com:443
management.azure.com:443
imageregistrygpei1358nqx.blob.core.windows.net:443
```
I didn't find `www.office.com` there, could you point me where it was used and required?

File: modules/configuring-firewall.adoc [hunk 4]
```diff
@@ -51,6 +51,10 @@ CDN hostnames, such as `cdn01.quay.io`, are covered when you add a wildcard entr
 |===
 |URL | Port | Function
 
+|`cloud.redhat.com`
+|443, 80
+|Required for Telemetry
```
Comment (gpei) [inline modules/configuring-firewall.adoc:56]: Installer QE can't ack this, we don't test Telemetry

File: modules/configuring-firewall.adoc (hunk 3 above)
Comment (Amrita42) [reply modules/configuring-firewall.adoc:112]: @gpei the customer case comment has :
www.office.com			443, 80		Required to access Azure services and resources. Review the Azure REST API Reference in the Azure documentation to determine the endpoints to allow for your APIs.

File: modules/configuring-firewall.adoc (hunk 4 above)
Comment (Amrita42) [reply modules/configuring-firewall.adoc:56]: I will remove this then?

Comment (gpei) [reply modules/configuring-firewall.adoc:56]: sorry, I don't know, we don't test this.

Comment (gpei) [reply modules/configuring-firewall.adoc:56]: sorry, I don't know, we don't test this.

File: modules/configuring-firewall.adoc (hunk 3 above)
Comment (gpei) [reply modules/configuring-firewall.adoc:112]: I don't know where it was used

Comment (dmunne) [reply modules/configuring-firewall.adoc:112]: @gpei the `www.office.com` was registered during an OpenShift 4.6.47 installation in a customer environment with a transparent proxy. I think it's part of the redirection in the Azure/Microsft API to retrieve the access token to complete the validation to `management.azure.com`. I could see some connections to `login.microsoftonline.com` and `login.office.com`.

**Comment by gpei** [comment]
LGTM

**Pull request merged**